src/main/java/net/imagej/tensorflow/util/UnpackUtil.java

```diff
@@ -140,6 +140,9 @@ private static void unZip(ZipInputStream zis, File output, LogService log, Statu
 			final String name = entry.getName();
 			log("Unpacking " + name, log, status);
 			final File outFile = new File(output, name);
+			if (!outFile.toPath().normalize().startsWith(output.toPath())) {
```

Comment:
CI fails with:
```
  [INFO] Caching TensorFlow models to /home/runner/work/imagej-tensorflow/imagej-tensorflow/./models
  [INFO] Unpacking imagenet_comp_graph_label_strings.txt
  Error:  java.lang.RuntimeException: Bad zip entry
  	at net.imagej.tensorflow.util.UnpackUtil.unZip(UnpackUtil.java:144)
  	at net.imagej.tensorflow.util.UnpackUtil.unZip(UnpackUtil.java:131)
  	at net.imagej.tensorflow.DefaultTensorFlowService.downloadAndUnpackResource(DefaultTensorFlowService.java:407)
  	at net.imagej.tensorflow.DefaultTensorFlowService.modelDir(DefaultTensorFlowService.java:384)
  	at net.imagej.tensorflow.DefaultTensorFlowService.loadGraph(DefaultTensorFlowService.java:153)
  	at net.imagej.tensorflow.demo.LabelImage.run(LabelImage.java:107)
```
So it appears we need a little manual adjustment for the zip slip guard and/or zip unpacking path prefix logic. I'll try to look into it when I can...

Reply:
If you find a bug in my fix, please let me know ASAP, I just sent this out to 100 other projects. 😬

Reply:
(Posting on behalf of @JLLeitschuh as he is currently rate-limited :D)
Supposedly, this should fix the issue:

```suggestion
			if (!outFile.toPath().normalize().startsWith(output.toPath().normalize())) {
```

Reply:
Thanks!